templates/p5_env.bt

```diff
@@ -64,35 +64,35 @@ typedef struct
 
 typedef struct
 {
-    SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field02<name = "Field Model Diffuse Red">;
-	f32 Field06<name = "Field Model Diffuse Green">;
-	f32 Field0A<name = "Field Model Diffuse Blue">;
-	f32 Field0E<name = "Field Model Diffuse Alpha">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
 	f32 Field12<name = "Field Model Ambient Red">;
 	f32 Field16<name = "Field Model Ambient Green">;
 	f32 Field1A<name = "Field Model Ambient Blue">;
 	f32 Field1E<name = "Field Model Ambient Alpha">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field22<name = "Field Model Specular Red">;
-	f32 Field26<name = "Field Model Specular Green">;
-	f32 Field2A<name = "Field Model Specular Blue">;
-	f32 Field2E<name = "Field Model Specular Alpha">;
+	f32 Field22<name = "Field Model Diffuse Red">;
+	f32 Field26<name = "Field Model Diffuse Green">;
+	f32 Field2A<name = "Field Model Diffuse Blue">;
+	f32 Field2E<name = "Field Model Diffuse Alpha">;
+    SetBackColor( MyRandom( 0xFFFFFFFF ) );
+	f32 Field32<name = "Field Model Specular Red">;
+	f32 Field36<name = "Field Model Specular Green">;
+	f32 Field3A<name = "Field Model Specular Blue">;
+	f32 Field3E<name = "Field Model Specular Alpha">;
     SetBackColor( MyRandom( 0xFFFFFFFF ) );
-	f32 Field32<name = "Field Model Emissive Red">;
-	f32 Field36<name = "Field Model Emissive Green">;
-	f32 Field3A<name = "Field Model Emissive Blue">;
-	f32 Field3E<name = "Field Model Emissive Alpha">;
+	f32 Field42<name = "Field Model Emissive Red">;
+	f32 Field46<name = "Field Model Emissive Green">;
+	f32 Field4A<name = "Field Model Emissive Blue">;
+	f32 Field4E<name = "Field Model Emissive Alpha">;
 	SetBackColor( MyRandom( 0xFFFFFFFF ) );
-    f32 Field42;
-	f32 Field46;
-	f32 Field4A;
-	f32 Field4E;
+    f32 Field52<name = "Field52">;
```

Comment:
Setting the name is not needed if it's the same as the name of the field.

Reply:
Done